tzif.zig

```diff
@@ -1120,22 +1120,278 @@ test "posix TZ string" {
     // 2021-11-07T02:00:00-06:00 (1st Sunday of the 11th month, MDT)
     try testing.expectEqual(stdoff, result.offset(1636272000).offset);
 
-    // e.g. Europe/Berlin; DST transition time at 2 am if DST off-->on, 3 am if DST on-->off
+    // IANA identifier: Europe/Berlin
     result = try parsePosixTZ("CET-1CEST,M3.5.0,M10.5.0/3");
     stdoff = 3600;
     dstoff = 7200;
-
     try testing.expectEqualSlices(u8, "CET", result.std_designation);
     try testing.expectEqual(stdoff, result.std_offset);
     try testing.expectEqualSlices(u8, "CEST", result.dst_designation.?);
     try testing.expectEqual(dstoff, result.dst_offset);
     try testing.expectEqual(PosixTZ.Rule{ .MonthNthWeekDay = .{ .month = 3, .n = 5, .day = 0, .time = 2 * std.time.s_per_hour } }, result.dst_range.?.start);
     try testing.expectEqual(PosixTZ.Rule{ .MonthNthWeekDay = .{ .month = 10, .n = 5, .day = 0, .time = 3 * std.time.s_per_hour } }, result.dst_range.?.end);
-
     // 2023-10-29T00:59:59Z, or 2023-10-29 01:59:59 CEST. Offset should still be CEST.
     try testing.expectEqual(dstoff, result.offset(1698541199).offset);
     // 2023-10-29T01:00:00Z, or 2023-10-29 03:00:00 CEST. Offset should now be CET.
     try testing.expectEqual(stdoff, result.offset(1698541200).offset);
 
-    // TODO : add more tests
+    // IANA identifier: America/New_York
+    result = try parsePosixTZ("EST5EDT,M3.2.0/02:00:00,M11.1.0");
+    stdoff = -18000;
+    dstoff = -14400;
+    try testing.expectEqualSlices(u8, "EST", result.std_designation);
+    try testing.expectEqual(stdoff, result.std_offset);
+    try testing.expectEqualSlices(u8, "EDT", result.dst_designation.?);
+    try testing.expectEqual(dstoff, result.dst_offset);
+    // // ---
+    // // 2020, leap year
+    // // FIXME : this does not work yet
+    // // transition std 2020-03-08T01:59:59-05:00 --> dst 2020-03-08T03:00:00-04:00
+    // try testing.expectEqual(stdoff, result.offset(1583650799).offset);
+    // try testing.expectEqual(dstoff, result.offset(1583650800).offset);
```

Comment:
Hmm... I'm thinking it would be good to split these out into a separate test block so that we can skip the test instead of commenting it out. E.g.:

```zig
test "PosixTZ 2020 leap year" {
    // FIXME : this does not work yet
    if (true) return error.SkipZigTest;
    try testing.expectEqual(stdoff, result.offset(1337).offset);
}
```
The `if (true)` is just so zig doesn't detect code after the return as dead code and error.